MVVMRxSwiftDemoTests/Sources/Specs/Data/Repositories/Authentication/AuthenticationRepositorySpec.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  AuthenticationRepositorySpec.swift
+//  MVVMRxSwiftDemoTests
+//
+//  Created by Chananchida F. on 7/23/21.
+//
+
+@testable import MVVMRxSwiftDemo
+import Quick
+import Nimble
+import RxSwift
+import RxNimble
+
+final class AuthenticationRepositorySpec: QuickSpec {
+
+    override func spec() {
+
+        var authenticationRepository: AuthenticationRepository!
+        var networkAPITokenMock: NetworkAPIMock!
```

Comment:
I would prefer to keep the name more general here. Because in some cases, we have more models (APIToken, User, ...) in `AuthenticationRepository`, not only `APIToken`.
```suggestion
        var networkAPIMock: NetworkAPIMock!
```